packages/playground/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "scripts": {
     "build": "stencil build --debug && cp src/_redirects www/_redirects",
-    "start": "stencil build --dev --watch --serve",
+    "serve": "stencil build --dev --watch --serve",
```

Comment:
@ebryn we'll have to update the CD script to use `serve` instead of `start` for the playground

Reply:
We can't keep it as `start`?